src/components/RuntimeManager.js

```diff
@@ -196,8 +197,8 @@ export default class RuntimeManager extends PureComponent {
     const createdDate = new Date(runtime.createdDate)
     const dateNotified = getDynamic(sessionStorage, `notifiedOutdatedRuntime${runtime.id}`) || {}
     const rStudioLaunchLink = Nav.getLink(appLauncherTabName, { namespace, name, application: 'RStudio' })
-    const app = getCurrentApp(apps)
-    const prevApp = getCurrentApp(prevProps.apps)
+    const galaxyApp = getCurrentAppForType(tools.galaxy.appType)(apps)
+    const prevGalaxyApp = getCurrentAppForType(tools.galaxy.appType)(prevProps.apps)
```

Comment:
perhaps a TODO here (maybe group these TODOS with some `generalize apps` tag or the like) to make this a list of apps. The intent being it would be nice to get the launch notification relatively for free when adding new apps!

Reply:
Can we add this the existing IA-2848 ticket? I agree it would be nice to get this support for Cromwell as well.

My changes didn't make the code any more app-specific than it was before. It's just that previously `app` was always a Galaxy app, and the code that used `app` showed Galaxy-specific messages and functionality. I renamed the variables so that it is more obvious what is going on.

Reply:
Yeah, I don't think it should go in the scope of this PR.

Maybe just a TODO for now. We would likely get to it when we add in other apps in future quarters, not with the analysis tab release which is what the epic that ticket is intended for.

Reply:
I created https://broadworkbench.atlassian.net/browse/BW-929.